src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -233,5 +233,10 @@ public override int GetHashCode()
                 Hash.Combine(base.GetHashCodeHelper(),
                 Hash.Combine((int)this.SpecifiedLanguageVersion, 0));
         }
+
+        public override ParseOptions WithLatestLanguageVersion()
+        {
+            return new CSharpParseOptions(LanguageVersion.Latest);
```

Comment:
This is wrong (losing other information inside the parse options you start with). See `WithFeatures` above for contrast.
Assuming we end adding such a compiler API, it needs corresponding API-level compiler tests.